internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -16,6 +16,12 @@ DeploymentConfiguration:
     Rollback: true
   MinimumHealthyPercent: {{ .DeploymentConfiguration.MinHealthyPercent }}
   MaximumPercent: {{ .DeploymentConfiguration.MaxPercent }}
+  {{- if .DeploymentConfiguration.RollbackAlarms }}
+  Alarms:
+    AlarmNames: {{ .DeploymentConfiguration.RollbackAlarms }}
```

Comment:
Just double checking: does  `.RollbackAlarms` - which is `[]*string` instead of `[]string` -  render properly here?

Reply:
it's now `[]*string` in `manifest` but `[]string` in template